docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloudquery-source.md

```diff
@@ -0,0 +1,98 @@
+---
+id: cloudquery-source
+title: CloudQuery Source
+sidebar_label: CloudQuery
+tags:
+  - cloud-to-cloud
+  - cloudquery
+description: Learn how to collect inventory from the AWS APIs and transform them into the CloudQuery schema and send it to Sumo Logic.
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/cloudquery/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/cloudquery/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/cloudquery/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/cloudquery-logo.png')} alt="cloudquery-icon" width="70" />
+
+The CloudQuery integration is used to pull inventory from the AWS APIs and transform them into the CloudQuery schema and send it to Sumo Logic.
+
+:::note
+This source is available in the [Fed deployment](https://hub.cloudquery.io/plugins/source/cloudquery/aws/v22.19.2/docs).
+:::
+
+## Data collected
+
+| Polling Interval | Data |
+| :--- | :--- |
+| 24 hours |  [Data service table data](https://developer.code42.com/api) |
+
+## Setup
+
+### Source configuration
+
+When you create an CloudQuery source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure a CloudQueryr Source:
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
+1. On the Collection page, click **Add Source** next to a Hosted Collector.
+1. Search for and select **CloudQuery**.
+1. Enter a **Name** for the source. The description is optional. 
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. **AWS Access Key ID**. Enter the Access Key ID collected from the [AWS Management Console](#vendor-configuration).
+1. **AWS Secret Access Key**. Enter the Secret Access Key collected from the [AWS Management Console](#vendor-configuration).
```

Comment:
The links on for "AWS Management Console" are broken because there is no "Vendor Configuration" section in the article.